### ISSUE TITLE
android 10. 'widget/programm keeps closing'

### ISSUE THREAD

**Issue by pieterjoanUvA** [ashutoshgngwr/10-bitClockWidget#24]
on a samsung skinned android 10 phone, xcover4s,
I can select the widget and view the placement area,

then effectively nothing shows.

after some stubborn retries, android notifies me that the app keeps closing.

first saw it on fdroid, then tried the paystore with the same effect.

this might keep my hobby and education alive ;-]

**Comment by ashutoshgngwr** [comment]
That's weird! I am also running it on Android 10 (Xiaomi Mi A3). Let me check!

**Comment by ashutoshgngwr** [comment]
Can you tell which app version are you using? 1.6-1 had a similar bug which I patched in 1.6-2. 1.6-2 is not yet available on F-Droid. Download it here instead - https://github.com/ashutoshgngwr/10-bitClockWidget/releases/download/v1.6-2/10-bitClockWidget-release.apk

**Comment by pieterjoanUvA** [comment]
I also just tested that one, and made some screenshots,
in the clock-till-battery_bar was a clock widget icon, with an extra status message.
![Screenshot_20201023-102910_Settings](https://user-images.githubusercontent.com/36740203/96975373-a6667e80-151a-11eb-8919-0d987374bd69.jpg)

![Screenshot_20201023-101919_Settings](https://user-images.githubusercontent.com/36740203/96975265-8636bf80-151a-11eb-85c5-b816c6ba1478.jpg)
![Screenshot_20201023-101909_One UI Home](https://user-images.githubusercontent.com/36740203/96975269-8767ec80-151a-11eb-9b05-ca83d2e899c2.jpg)

**Comment by ashutoshgngwr** [comment]
Thanks for the effort. ADB wouldn't help because the APK is obfuscated and the mapping file didn't get uploaded anywhere. In layman's terms, I messed up.

I think I have a general idea of the issue. I'll try to fix it and create a release ASAP. If that release is not able to fix it, I'll ensure that we're at least able to capture the deobfuscated crash logs using ADB.

**Comment by pieterjoanUvA** [comment]
Thanks in advance for the time spent, to keep up with all new Api's and Google always-changin'-guidelines.

:-]

**Comment by ashutoshgngwr** [comment]
I have created a new release with a potential fix. Let me know if it works.

Download - https://github.com/ashutoshgngwr/10-bitClockWidget/releases/download/v1.6-3/10-bitClockWidget-release.apk

If it still crashes, run following ADB command after the crash and copy-paste its output here.

```shell
adb logcat "*:E" --pid="$(adb shell pidof -s com.github.ashutoshgngwr.tenbitclockwidget)"
```

If this doesn't produce an  output longer than a single line, use

```shell
adb logcat --pid="$(adb shell pidof -s com.github.ashutoshgngwr.tenbitclockwidget)"
```

**Comment by pieterjoanUvA** [comment]
this works, greatly appreciated. 

![Screenshot_20201023-123754_One UI Home](https://user-images.githubusercontent.com/36740203/96994282-c99a2980-152c-11eb-9305-f2d20c3ea606.jpg)

**Comment by ashutoshgngwr** [comment]
Thanks! You can hide the notification for a seamless experience.

**Comment by pieterjoanUvA** [comment]
yes, I did notice that one, and the explanation was clear.
the notification options are usually a bit unclear and/or hard to find or maintain. 
I personally like your solution to help one to get rid of the probably good intentions to show it. 
since android 9 I have had issues with those and some long loved little apps, on seriously underpowered phones.
eg. single core 600mhz, android 4.

and those little but very nice utils often don't grow nicely with new requirements. 

the quick response and fixing , is a great one.

kind regards,
Pieter Joan
